ghost/ghost/src/http/guards/admin-api-authentication.guard.ts

```diff
@@ -0,0 +1,100 @@
+import {
+    Injectable,
+    CanActivate,
+    ExecutionContext,
+    Inject
+} from '@nestjs/common';
+import {Request, Response} from 'express';
+import {Actor} from '../../common/types/actor.type';
+import ObjectID from 'bson-objectid';
+
+// Here we extend the express Request interface with our new type
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
If we are actually using the namespacing - worth relaxing the eslint rule globally? Otherwise leave a comment about why and the benefit of namespacing in this situation?